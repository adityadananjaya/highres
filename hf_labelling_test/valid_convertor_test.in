python3 label_program.py yolotxt_to_voc ./hf_labelling_test/convertor_test/