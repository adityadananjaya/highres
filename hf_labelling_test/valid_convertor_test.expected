[['(-3.999, -3.8, 4.001, 4.2)', '(-6.99, -8.7, 9.01, 11.5)', '(-4.0, -5.800000000000001, 8.0, 13.8)', '(-7.0, -10.0, 21.0, 30.0)', '(-99.0, -99.0, 101.0, 101.0)', '(-900.0, -900.0, 1100.0, 1100.0)'], ['(-0.20890600000000004, -0.07924999999999993, 1.172344, 1.3473060000000001)'], ['(-1.0, -1.0, 3.0, 3.0)', '(-2.0, -3.0, 3.0, 4.0)', '(-3.8, -5.7589999999999995, 4.2, 6.239)']]
